### ISSUE TITLE
Error Occured: Failed to call api method

### ISSUE THREAD

**Issue by AncientEntity** [pygame-web/pygbag#165]
When building and testing a pygbag game I'm currently getting this error popup: "Error occurred: Failed to call api method: could not connect to provider anyweb-to-anyweb-unprivileged-api-v1"

I wasn't getting it before, and the build was working, then i went ahead and rebuilt it since I haven't touched it in a few months and I get this error. I also build an empty file and got the error too.

**Comment by pmp-p** [comment]
Sorry we cannot support external api you are using in your games if you do not provide full source,assets + some credentials for those api services.

**Comment by AncientEntity** [comment]
> Sorry we cannot support external api you are using in your games if you do not provide full source,assets + some credentials for those api services.

Doesn't matter what I try building, If I build the example main.py in the README.md of pygbag I get the same issue. Is it possible something with my installed python packages could be causing the issue? I have 2 versions of python installed 3.11/3.12 maybe they're conflicting? (Pygbag only installed for 3.11) But I did try in a virtual environment and still had issues.

Code I attempted to build: [main.txt](https://github.com/pygame-web/pygbag/files/15081709/main.txt)
